problemsets-mat223/modules/module9-exercises.tex

```diff
@@ -1,17 +1,62 @@
 \begin{exercises}
 	\begin{problist}
-		\prob  Example problem with parts
+		\prob For each transformation
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
-		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
+			\item if it is linear then prove it, otherwise
+			show why it is not.
+			\item find the corresponding matrix.
 			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
+				\item Let $\mathcal S:\R^2\to\R^2$ be defined by
+				$\mathcal S(\vec x) = \mat{3x\\-2y}$
+				\item Let $\mathcal T:\R^2\to\R^2$ be defined by
+				$\mathcal T(\vec x) = \mat{2x\\y+1}$
+				\item Let $\mathcal U:\R^2\to\R^2$ be defined by
+				$\mathcal U(\vec x) = \mat{0\\0}$
+				\item Let $\mathcal V:\R^3\to\R^3$ be defined by
+				$\mathcal V(\vec x) = \mat{x^2\\y^2\\z^2}$
+				\item Let $\mathcal W:\R^3\to\R^3$ be defined by
+				$\mathcal W(\vec x) = \mat{\frac{1}{2}x\\y\\z-2y}$
+				\item Let $\mathcal X:\R^3\to\R^3$ be defined by
+				$\mathcal X(\vec x) = \mat{z\\0\\z}$
+				\item Let $\mathcal Y:\R^3\to\R^2$ be defined by
+				$\mathcal Y(\vec x) = \mat{x\\2x}$
+				\item Let $\mathcal Z:\R^2\to\R^4$ be defined by
+				$\mathcal Z(\vec x) = \mat{2y\\0\\0\\-x+y}$
```

Comment:
Replace some of these with transformations described in words, like "T:R^2\toR^2, where T reflect vectors over the line y=x", etc. You've got reflections, rotations, translations, projections, dilation which can all be described in words :-). (You can even throw in a hard composition, like translate \circ rotation \circ untranslate)